### DISCUSSION TITLE
Grouped Bar Chart Based on Distinct Fields

### DISCUSSION THREAD

**Discussion by theduke** [vega/vega-lite#7294] [Q&A] [answered]
Hi there.

From reading the documentation, I couldn't figure out how to create grouped bar charts based on two distinct fields in the data.

Just as an example:

```json
"data": [
  {"countA": 100, "countB": 7},
  ...
]
```

**Comment by domoritz** [comment, marked as answer]
You can use the [fold transform](https://vega.github.io/vega-lite/docs/fold.html) to convert the format you have to the expected long form/tidy form.
